README.md

```diff
@@ -1,2 +1,62 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # Awesome-Books
-A basic website that allows users to add/remove books from a list using JavaScript objects and arrays. 
+A website that allows users to add/remove books from a list using JavaScript objects and arrays. 
+
+> Description the project.
+Awesome Books is a website that allows users to add/remove books from a list using JavaScript objects and arrays, we hope that you enjoy using it as we did building it.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- A lot of Coffee
+
+## Live Demo (if available)
+
+[Coming Soon!](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
+
+
+## Getting Started
+
+###To get a local copy up and running follow these simple example steps:
+
+- Click the Code button in the top right corner
+- Download ZIP
+- Unzip the file: hello-microverse.zip
+- Open index.html with your web browser
+
+To get a local copy up and running follow these simple example steps.
+
+
+## Authors
+
+👤 **Daniel**
+
+- GitHub: [@dgonzalesi](https://github.com/dgonzalesi/)
+- Twitter: [@dgonzalesi](https://twitter.com/dgonzalesi/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/daniel-g-sierra-60472719/)
+
+👤 **Stanley**
+
+- GitHub: [@githubhandle](https://github.com/StarMindz)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to add a issue here: [issues page](https://github.com/StarMindz/Awesome-Books/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To all Tomorrowland performers that give us music while we coding.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯